s4e-web/src/app/views/settings/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="breadcrumb">
+  <span i18n>Tutaj jesteś: </span>
+  <span>
+    <ng-container *ngFor="let breadcrumb of breadcrumbs$ | async; index as index$">
+      <ng-container [ngSwitch]="index$">
+        <ng-container *ngSwitchCase="(breadcrumbs$ | async).length - 1">
+          {{ breadcrumb.label }}
+        </ng-container>
+        <ng-container *ngSwitchDefault>
+          <a [routerLink]="breadcrumb.url" queryParamsHandling="preserve">{{ breadcrumb.label }}</a> >
+        </ng-container>
+      </ng-container>
+    </ng-container>
```

Comment:
You can do it in a much simpler way by using `first` and `last` (these are available in the same way as `index`

```
<ng-container *ngFor="let breadcrumb of breadcrumbs$ | async; last as last$">
```

`first` / `last` are normal `boolean` variables

Reply:
It isn't much simpler, but it is good to know it :)